maestro/server/Makefile

```diff
@@ -21,5 +22,7 @@ deploy:
 		--set image.base=${IMAGE_BASE} \
 		--set image.tag=${IMAGE_TAG} \
 		--set database.containerizedDb=${USE_CONTAINERIZED_DB} \
-		--set database.ssl='${USE_DATABASE_SSL}'
+		--set database.ssl='${USE_DATABASE_SSL}' \
+		$(if $(filter false,$(USE_CONTAINERIZED_DB)),--set database.host=$${DATABASE_HOST}) \
```

Comment:
Is this some fallback for other environments?

Reply:
i cleaned this up a bit, but yes the db host is either a static string for container DBs or an azure DB host